### DISCUSSION TITLE
DNS amplification attack prevention

### DISCUSSION THREAD

**Discussion by Waldorf3** [AdguardTeam/AdGuardHome#5912] [Q&A]
On https://adguard.com/en/blog/adguard-home-on-public-server.html it says:

> Many people scan the Internet for public DNS servers and try to exploit them for DNS amplification attacks. AdGuard Home has a reasonable default rate limit, so this is not a severe issue, but if you don't want to see unrelated IP addresses in the statistics, **you may want to restrict the access.**

... but I don't find any suggestions how we can do that?

**Comment by ainar-g** [comment]
See _Settings → DNS settings → Access settings._
